mkdocs.yml

```diff
@@ -184,42 +191,53 @@ nav:
     - Logging: 'concepts/logging.md'
     - Distillation: "concepts/distillation.md"
     - Union: 'concepts/union.md'
+    - Unions: 'concepts/unions.md'
+    - Validation: 'concepts/validation.md'
     - Alias: 'concepts/alias.md'
     - Enums: 'concepts/enums.md'
     - Type Adapter: 'concepts/typeadapter.md'
     - Templating: 'concepts/templating.md'
-  - Hub:
-    - Introduction to Instructor Hub: 'hub/index.md'
-    - Structured Outputs with Vertex AI: 'hub/vertexai.md'
-    - Structured Outputs with Ollama: 'hub/ollama.md'
-    - Structured Outputs with llama-cpp-python: 'hub/llama-cpp-python.md'
-    - Structured Outputs with Together: 'hub/together.md'
-    - Structured Outputs with Anyscale: 'hub/anyscale.md'
-    - Structured Outputs with Groq: 'hub/groq.md'
-    - Structured Outputs with Mistral: 'hub/mistral.md'
-    - Structured Outputs with Cohere: 'hub/cohere.md'
-    - Classification with Structured Outputs: 'hub/single_classification.md'
-    - Bulk Classification with Structured Outputs: 'hub/multiple_classification.md'
-    - Extracting Tables with Structured Outputs: 'hub/tables_from_vision.md'
-    - Creating Pandas DataFrames with Structured Outputs: 'hub/pandas_df.md'
-    - Bulk Async Classification with LangSmith: 'hub/batch_classification_langsmith.md'
-    - Extracting Action Items with Structured Outputs: 'hub/action_items.md'
-    - Implementing Partial Streaming Responses: 'hub/partial_streaming.md'
-    - Extracting Contact Information with Structured Outputs: 'hub/extract_contact_info.md'
-    - Generating Knowledge Graphs with Structured Outputs: 'hub/knowledge_graph.md'
-    - Extracting Relevant Clips from YouTube Videos: "hub/youtube_clips.md"
-    - Building Knowledge Graphs with Structured Outputs: 'tutorials/5-knowledge-graphs.ipynb'
+  - Integrations:
+    - Overview: 'integrations/index.md'
+    - Anyscale: 'integrations/anyscale.md'
+    - Anthropic: 'integrations/anthropic.md'
+    - Azure OpenAI: 'integrations/azure.md'
+    - Cerebras: 'integrations/cerebras.md'
+    - Cohere: 'integrations/cohere.md'
+    - Fireworks: 'integrations/fireworks.md'
+    - Google: 'integrations/google.md'
+    - Groq: 'integrations/groq.md'
+    - LiteLLM: 'integrations/litellm.md'
+    - llama-cpp-python: 'integrations/llama-cpp-python.md'
+    - Mistral: 'integrations/mistral.md'
+    - Ollama: 'integrations/ollama.md'
+    - OpenAI: 'integrations/openai.md'
+    - Together: 'integrations/together.md'
+    - Vertex AI: 'integrations/vertex.md'
+    - Single Classification: 'integrations/single_classification.md'
```

Comment:
the non llm clients should be part of hub 

- Single Classification: 'integrations/single_classification.md'
    - Bulk Classification: 'integrations/multiple_classification.md'
    - Extracting Tables: 'integrations/tables_from_vision.md'
    - Creating Pandas DataFrames: 'integrations/pandas_df.md'
    - Bulk Async Classification with LangSmith: 'integrations/batch_classification_langsmith.md'
    - Extracting Action Items: 'integrations/action_items.md'
    - Implementing Partial Streaming Responses: 'integrations/partial_streaming.md'
    - Extracting Contact Information: 'integrations/extract_contact_info.md'
    - Generating Knowledge Graphs: 'integrations/knowledge_graph.md'
    - Extracting Relevant Clips from YouTube Videos: "integrations/youtube_clips.md"